temoa/extensions/economies_of_scale/components/cost_fixed_eos.py

```
@@ -0,0 +1,307 @@
+    return cap == cap_eos
+
+
+# --- Calculating costs ---
+def cost_fixed_eos_segment_cost(
+    model: EOSModel,
+    r: Region,
+    p: Period,
+    t: Technology,
+    n: int,
+    capacity: ExprLike,
+    binary: VarData | bool = True,
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)